src/test/java/TestDataHandler.java

```diff
@@ -0,0 +1,67 @@
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mockito;
+import training.supportbank.DataHandler;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.math.BigDecimal;
+import java.util.*;
+
+import static org.assertj.core.api.Assertions.*;
+
+public class TestDataHandler {
+    private static List<Map<String, String>> expectedArray = new ArrayList<>();
+    private static Map<String, BigDecimal> expectedBalancesMap = new LinkedHashMap<>();
+
+    @Before
+    public void populateExpectedOutputs() {
+        expectedArray.clear();
```

Comment:
Try to avoid modifying and resetting values used in tests - it's easy to forget, and it makes it hard to track down bugs in the tests.

Ideally, each test should use its own input data and expected values. If that's verbose, it usually means you're testing too much and should simplify the tests or the code - we can have a chat about this if you're not sure.

Reply:
Resolved by changing to `@BeforeClass`